app/models/graders/junit_grader.rb

```diff
@@ -138,7 +143,13 @@ def send_build_request_to_orca
           URI.parse("#{orca_url}/api/v1/grader_images"),
           orca_image_build_config
         )
-        handle_image_build_attempt body['message'], status_code
+        if body['errors'].nil?
```

Comment:
Use `body['errors'].blank?`, which will handle both `nil` and also empty arrays `[]`